concourse/setup.py

```diff
@@ -39,6 +39,9 @@
     ConcourseConfig,
     ConcourseApiVersion,
 )
+from model.webhook_dispatcher import (
+    WebhookDispatcher
```

Comment:
should be named WebhookDispatcherC(on)?f(i)?g for the sake of consistency

Reply:
Done